### PR TITLE
Security 2020: markdown content

### PR DESCRIPTION
**Pull request by tomvangoethem**
Progress for #906

### CODE REVIEW

File: src/content/en/2020/security.md [hunk 1]
```diff
@@ -22,3 +22,605 @@ queries: 11_Security
...
+
+In this chapter, we explore the current state-of-practice for security on the Web. By analyzing the adoption of various security features in depth and at a large scale, we gather insights on the different ways that website owners apply these security mechanisms, driven by the incentive to protect their users. However, we not only look at the adoption of security mechanisms in individual websites. We analyze how different factors, such as the technology stack that is used to build the site, affect the prevalence of security headers, and thus improve overall security. Furthermore, because ensuring that a website is secure requires a holistic approach covering many different facets, we also evaluate other aspects, such as the patching practices for various widely used web technologies. Finally, we report on how security on the web has evolved in the last year and provide an outlook for what is yet to come.
+
+{# TODO do we want to keep the Methodology section? #}
+
+{# TODO Barry promised to write the TLS section #}
```
Comment (rviscomi) [inline src/content/en/2020/security.md:35]: cc @bazzadp 👀

Comment (bazzadp) [reply src/content/en/2020/security.md]: Made a start but need some sleep...

Comment (bazzadp) [reply src/content/en/2020/security.md]: Done!

**Comment by rviscomi** [comment]
@tomvangoethem what are the remaining action items before you're comfortable publishing this? I see a few TODOs but would you be ok launching without them?

**Comment by tomvangoethem** [comment]
The placeholders for the images still need to be updated (@barrypollard said he'll be doing that). For the rest it's mainly copyediting I think.
I'm fine with launching once the images are there.

**Comment by rviscomi** [comment]
Sounds great, thank you to the whole team for getting this ready for launch!

**Comment by bazzadp** [comment]
I’ve uploaded the figures and done a round of editing on most of it down to line 729 (Security mechanism adoption). I'd like to finish some more editing tomorrow but will merge this now and open a new PR for that.

I’ve staged the current version here: https://20201209t033020-dot-webalmanac.uk.r.appspot.com/en/2020/security.

**Pull request merged**